docs/source/advanced_usage.rst

```diff
@@ -299,19 +293,15 @@ programs, called ``ParametricPrograms``:
     from pyquil.parametric import ParametricProgram
     par_p = ParametricProgram(rotator) # This produces a new type of parameterized program object
 
-The parametric program ``par_p`` now takes the same arguments as ``rotator``:
-
-.. code:: python
-
-    print(par_p(0.5))
-
-.. parsed-literal::
+Please consider simply using a Python function for the above functionality::
```

Comment:
I think this could be made a little more explicit: `This object has been deprecated and removed. If you want the above functionality, you can simply use a Python function:`. (Otherwise, if I had been using the parametric program, and I really loved it, I would read this and think that _maaaybe_ I could somehow still use it.)

Reply:
done